scripts/annotate.py

```diff
@@ -1,11 +1,22 @@
+from ast import arg
```

Comment:
I think there are still a couple unused imports (e.g. arg and require)

Reply:
Ugh... I honestly do not know how these automatically get imported...